tests/test_latent_algebra.py

```diff
@@ -27,29 +27,34 @@ def test_latent_py_ps_inv():
     ps, py, inv = latent_algebra.compute_ps_py_inv_noise_matrix(s, nm)
     assert all(abs(np.dot(inv, ps) - py) < 1e-3)
     assert all(abs(np.dot(nm, py) - ps) < 1e-3)
```

Comment:
A future version of Pytest will not support return statements, 
hence separating this into separate functions